src/main/java/com/bookmarkmanager/controller/BookmarkController.java

```diff
@@ -0,0 +1,44 @@
+package com.bookmarkmanager.controller;
+
+import com.bookmarkmanager.model.Bookmark;
+import com.bookmarkmanager.service.BookmarkService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/bookmarks")
+public class BookmarkController {
+
+    @Autowired
+    private BookmarkService bookmarkService = BookmarkService.getInstance();
```

Comment:
@Autowired is used for creating the instance for you, why are you implementing Singleton here?